packages/context/src/binding.ts

```diff
@@ -267,9 +267,9 @@ function toValueFactory<T = unknown>(
  * Check if the factory is a value factory provider class
  * @param factory - A factory function or a dynamic value provider class
  */
-function isDynamicValueProviderClass(
+export function isDynamicValueProviderClass<T = unknown>(
```

Comment:
dynamic value provider classes was already supported is it ? (not able to find any examples / docs for it)

Reply:
See docs in this PR and https://github.com/strongloop/loopback-next/blob/master/docs/site/Binding.md#a-factory-function.